### PR TITLE
Testkube Doc: refine  test doc for content.type

### PR DESCRIPTION
**Pull request by gberche-orange**
## Pull request description 

Doc: refine schema for content.type

Extracted from https://github.com/kubeshop/testkube/blob/cca3c13629078891f269f691e1050ae5adb99d5a/docs/docs/test-types/prebuilt-executor.md?plain=1#L60-L65

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: charts/testkube-operator/templates/tests.testkube.io_tests.yaml [hunk 1]
```diff
@@ -428,7 +428,12 @@ spec:
                           type: string
                       type: object
                     type:
-                      description: test type
+                      description: |
+                        type of sources a runner can get data from.
+                            string: String content (e.g. Postman JSON file).
+                            file-uri: content stored on the webserver.
+                            git-file: the file stored in the Git repo in the given path.
+                            git-dir: the entire git repo or git subdirectory (Testkube does a spatial checkout to limit traffic in the case of monorepos).
```
Comment (vsukhin) [inline charts/testkube-operator/templates/tests.testkube.io_tests.yaml:436]: actually, we also support `git` type, it automatically detects file or dirs

Comment (gberche-orange) [reply charts/testkube-operator/templates/tests.testkube.io_tests.yaml]: thanks, I gave it a try into 12313d90a70804887a7c456dbb9210112fb16a9f does it properly captures current behavior ?

Comment (vsukhin) [reply charts/testkube-operator/templates/tests.testkube.io_tests.yaml]: internally, we already trreat everything as git. yes, the idea is to support monorepo as well

Comment (gberche-orange) [reply charts/testkube-operator/templates/tests.testkube.io_tests.yaml]: thanks,

> BTW, is it intended that the sparse checkout leaves the /data/repo with root of the git repo and not the subdir as new root ?

 sorry I had missed the `workingDir` field which answers my question

```
                        workingDir:
                          description: if provided we checkout the whole repository
                            and run test from this directory
                          type: string
``

**Pull request merged**